src/filter/Filter.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+export default class Filter extends Component {
+  render() {
+    return (
+      <div>
+        <form>
+          <h1>Find the Best Specialist</h1>
+          <input
+            type="text"
+            className="search-field name"
+            placeholder="Name..."
+            onChange={(event) => this.setState({ name: event.target.value })}
```

Comment:
this.setState({ name: event.target.value }) can cause error since there is no state **name** in this component

Reply:
i shall remove that since i was only working on the UI and someone else was to handle the funcionality. Thanks for the review.